### DISCUSSION TITLE
overview highlights on CherryNetwork profile

### DISCUSSION THREAD

**Discussion by Anyaoha** [CherryNetwork/Cherry-Node#126] [Show and tell]
Hello I have an idea of creating a github hightlight profile for the CherryNetwork. This will give individuals a snapshot on top projects, top languages with original source to facilitate collaborations from the wider community who come across the project. If this is something that interests you. I can share a markdown file that can be merged to the doc repo (which I believe should be created).

**Comment by charmitro** [comment]
Hey @Anyaoha! 
Sounds lovely! Prepare the markdown file and share it with us. 
After that we'll see what we can bring to the table with a doc repo.

Thank you for your interest!

**Reply by Anyaoha** [reply]
I have created a READme that will be pasted on top of the Cherry Network repo for users to easily find out what's going on with the project rather than tediously going through all the repos.

@charmitro I'm requesting that someone within the org that has write access create a new ".github" repo with a profile path that contains a README.md file so I can PR the doc.

Find the md file here: https://github.com/Anyaoha/CherryNetwork-profile

**Reply by charmitro** [reply]
There ya go @Anyaoha ! https://github.com/CherryNetwork/.github
Apologies for being late! 🚀

**Reply by Anyaoha** [reply]
That's alright. Sending in the PR.

**Comment by Anyaoha** [comment]
@charmitro PR created. Tell me how it looks on the CherryNetwork page :)

**Reply by charmitro** [reply]
Thank you sir! We'll let the big gun review it and reach back!

**Reply by Anyaoha** [reply]
You're welcome. I like to put my money where my mouth is. Love the initiative within the project
